test/generators/ModelGenerators.scala

```diff
@@ -25,6 +25,13 @@ import uk.gov.hmrc.http.HttpVerbs._
 trait ModelGenerators {
   self: Generators =>
 
+  implicit lazy val arbitrarySubmissionState: Arbitrary[SubmissionState] = Arbitrary {
+    val validStates = Seq("notSubmitted", "submitted", "rejectedPendingChanges", "rejectedAndResubmitted")
```

Comment:
Could use the toString definitions here, just so you're not doubling up on static strings